### DISCUSSION TITLE
Idea

### DISCUSSION THREAD

**Discussion by esterer** [Whyzanet/WhyzaGC#3] [Ideas]
I wired in test point 1 as an interrupt and deleted the serial interface. It's less complicated.

**Comment by Whyzanet** [comment]
IThe main reason I didn't choose this path is I lacked the necessary test products at the time to do the required testing for different CPM (high) values. Since then I have obtained some test products which makes this moot. 

I was also happy to trust MightyOhms coding more than mine in regards to higher count accuracy.

Recently also this relevant unresolved thread was posted at radmon.org forums relating to higher counts, which is interesting.

https://radmon.org/index.php/forum/homebrew-counters-electronic-design-and-building/1291-struggling-with-esp32-counts-too-many-interrupt-issue

**Comment by esterer** [comment]
10-4. Radmon code has a few errors but I can't register on their website. Google is their friend now.
________________________________

**Reply by Whyzanet** [reply]
yes they have noted registration issues and a workaround at this thread

https://radmon.org/index.php/forum/radmon-org-news/1288-verification-email-issues-for-new-registrations

I would be intrigued to know what the issue is with their code.

**Comment by esterer** [comment]
Will post when resolved. Cheers,
John
________________________________

**Comment by esterer** [comment]
Thanks for the info. I posted some working interrupt code.
________________________________

**Comment by DemBones78** [comment]
this is a really cool project, have you used Home Assistant?   and ESP home?
i want to do this but also use ESP HOME to beable to put this in as a sensor in HomeAssistain

**Reply by Whyzanet** [reply]
No I haven't.

I will take a look when I get some time.

**Reply by Whyzanet** [reply]
> this is a really cool project, have you used Home Assistant? and ESP home? i want to do this but also use ESP HOME to beable to put this in as a sensor in HomeAssistain

I have added basic MQTT support in version 4. 
I will look at ESP Home maybe next
